VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: 


BO_ 1008 SGAMP1_data1000Hz: 3 Vector__XXX
 SG_ SGAMP1_ambientTemp : 23|8@0- (1,0) [0|0] "" Vector__XXX
 SG_ SGAMP1_outputVoltage : 7|16@0- (7.62939453125e-05,2.5) [-32768.0|32767.0] "V" Vector__XXX

BO_ 1009 SGAMP2_data1000Hz: 3 Vector__XXX
 SG_ SGAMP2_ambinetTemp  : 23|8@0- (1,0) [0|0] "" Vector__XXX
 SG_ SGAMP2_outputVoltage  : 7|16@0- (7.62939453125e-05,2.5) [-32768.0|32767.0] "V" Vector__XXX

BO_ 1010 SGAMP3_data1000Hz: 3 Vector__XXX
 SG_ SGAMP3_ambientTemp  : 23|8@0- (1,0) [0|0] "" Vector__XXX
 SG_ SGAMP3_outputVoltage  : 7|16@0- (7.62939453125e-05,2.5) [-32768.0|32767.0] "V" Vector__XXX

BO_ 1011 SGAMP4_data1000Hz: 3 Vector__XXX
 SG_ SGAMP4_ambientTemp  : 23|8@0- (1,0) [0|0] "" Vector__XXX
 SG_ SGAMP4_outputVoltage  : 7|16@0- (7.62939453125e-05,2.5) [-32768.0|32767.0] "V" Vector__XXX





BA_DEF_ BO_  "GenMsgCycleTime" INT 0 65535;
BA_DEF_DEF_  "GenMsgCycleTime" 0;
BA_ "GenMsgCycleTime" BO_ 1008 100;
BA_ "GenMsgCycleTime" BO_ 1009 100;
BA_ "GenMsgCycleTime" BO_ 1010 100;
BA_ "GenMsgCycleTime" BO_ 1011 100;




